main.py

```diff
@@ -1,40 +1,55 @@
 #!/usr/bin/env python
```

Comment:
we are desperately looking for some tests but cannot find any, do you know where they are?:D

Reply:
we also did not write any tests ourselves yet - how would you go about it? comparing random walker to random walker does not really make any sense.. 
maybe a check if a point / line intersects an obstacle would make sense but how would you test the other parts? test the functions seperately?